### ISSUE TITLE
Breeding Calculations Don't Match When bUseSinglePlayerSettings=True

### ISSUE THREAD

**Issue by saluce65** [thommcgrath/Beacon#280]
While testing the breeding stats on a non-dedicated server using local config files, I was testing to see if the numbers in-game match with the numbers shown in Beacon. With modifiers all set to `1`, I bred two dodos together. Once the female laid an egg, her time to mate was set for 4 hours, when the expected value was 18h-48h. When I checked my settings, I saw that `bUseSinglePlayer` was set to `True`.

After turning the setting to `False`, I bred another female; after laying an egg, her time to mate was 24h, as expected.

I tested this in Ark: Survival Ascended project. I do not know if it applies to Ark: Survival Evolved.

**Comment by saluce65** [comment]
I see the issue here. I now see the `2x Rates` button but was unaware that it was a dropdown menu that would toggle between Normal and Single Player stats. It seems like there's enough room there to turn those two items into radio-style toggle buttons, but I can also see why you wouldn't want to take up screen space for a Single Player button when the vast majority of users are using it for servers that won't have that option enabled.

Perhaps there might be a way to automatically switch the mode whenever the flag is set in the General Settings and when the project is first loaded? Or, at least, set the mode when the tab is activated based on the Single Player setting?

**Comment by thommcgrath** [comment]
So I hadn't really dug into this for ASA yet, so the numbers could just simply be wrong. Initial tests suggest that ASA creatures have vastly shorter mature times, for example.

Automating this is harder than you would expect. One of the problems is "Use Single Player Settings" could be in the indeterminate state. Then there's no way to know if it's on or off and we'd need the option anyway. And then if somebody turns it on in Breeding Multipliers, does that mean they want it on for their whole server? What about the other events?

The edge cases end up being more confusing than just leaving it as a reference-only option.

**Comment by saluce65** [comment]
Perhaps its the icon on the button itself that is confusing. When I looked at it, I was seeing "2x rates", which, when placed next to the Auto Imprint button, made me thing it was a button to double the normal rates. I'm trying to think of a way to rename the button to make the intent more clear without making the button too long (I mean, "Single Player/Event Rates" is too wordy, but "Special Rates" doesn't necessarily convey that these apply to Single Player config) and what kind of icon to use, but I'm drawing a blank. 

Maybe I could give myself some time to think, search through FontAwesome or other icon providers and see if anything strikes me as a better option, if you're interested.
